portfolio/src/main/webapp/script.js

```diff
@@ -114,21 +133,42 @@ document.addEventListener('DOMContentLoaded', function() {
  */
 function toggleForm() {
   const form = document.getElementById('recommendation-form');
+  const imageForm = document.getElementById('datastore-form');
   if (form.style.display == 'block') {
```

Comment:
Use === instead of == to avoid unintended type coercion.

Reply:
oh yes! okay!